### ISSUE TITLE
Suggestion to the issues

### ISSUE THREAD

**Issue by ck1988** [zhuowei/MCPELauncher#1317]
This is to the creator of blocklauncher I have seen and read the issues that people are having with the app such as white screen or the app not opening at all to myself it seams that there is a slight issue with the new update for mcpe 0.15.+ and my suggestion is that you pull up a backed up copy of the older versions of blocklauncher code that work and the new 0.15.+ update code it seems that there may be a part of the code missing or something extra that is interfering with the new update in the code that is being unnoticed and no one is seeing it. And I would like to say that it is a hard thing to do what the great creator of blocklauncher doing to make sure that you have a app that works to the fullest extent permitted by the device or devices that you are using so just be patient with the app and it's creator or creator's they do have a personal life away from the work they do.

**Comment by zedman2000** [comment]
@ck1988 that is kind of you to make such a plea.  Unfortunatly, if you look at all the 100's of repeats, they are not reading issues.  Just posting and then holding their hands out saying "gimme".  So sad.  I will say that a 0.15 version is not out and that they are using BL 1.12.8, which is for 0.14.3.  This is why they have the "white screen".  The betas do not do this "white screen" everyone keeps complaining about...why?  It is 0.15 compatible!  :-)

**Comment by ladyd99** [comment]
I agree, sometimes people can be such whiners. I have kit kat 4.4 and lollipop 5.1 which both crash on startup. I wish there was something I could do to help zhuowei I feel sad for him. This update must be causing him a lot of stress. and as I have stated on twitter to all the thankless complainers..sit back and be patient.  Good things come to those who wait.

**Comment by ladyd99** [comment]
That isnt good for the dev.  😂
